spot-client/src/common/app-state/remote-control-service/selectors.js

```diff
@@ -72,6 +86,20 @@ export function isConnectionPending(state) {
     return Boolean(connect && connect.requestState === asyncActionRequestStates.PENDING);
 }
 
+/**
+ * A selector which returns whether or a not a raise hand change command
+ * is currently in flight.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {boolean}
+ */
+export function isRaiseHandPending(state) {
+    const handRaised = state.remoteControlService.handRaised;
+
+    return Boolean(
+        handRaised && handRaised.requestState === asyncActionRequestStates.PENDING);
```

Comment:
```suggestion
        handRaised?.requestState === asyncActionRequestStates.PENDING);
```

I know the previous code didn't use optional chaining... it didn't exist back then!